### DISCUSSION TITLE
blog/10-%E4%BA%94%E6%9C%88%E7%AC%AC%E4%B8%89%E5%91%A8%E5%B0%8F%E8%AE%B0/

### DISCUSSION THREAD

**Discussion by giscus** [trevortylerlee/astro-micro#8] [Announcements]
# blog/10-%E4%BA%94%E6%9C%88%E7%AC%AC%E4%B8%89%E5%91%A8%E5%B0%8F%E8%AE%B0/

记录了这一周主要发生的事情。

https://blog.writely.xyz/blog/10-%E4%BA%94%E6%9C%88%E7%AC%AC%E4%B8%89%E5%91%A8%E5%B0%8F%E8%AE%B0/

**Comment by flashlab** [comment]
为什么消防员可以和程序员产生联系 :open_mouth:
